lib/src/components/NxVulnerabilityIndicator/types.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { WeakValidationMap } from 'react';
+import * as PropTypes from 'prop-types';
+
+import {
+  VulnerabilityScore,
+  VulnerabilitySeverityRating,
+  allVulnerabilitySeverityRatings
+} from '../../util/vulnerabilitySeverityRatings';
+
+export interface Props {
+  severityRating?: VulnerabilitySeverityRating;
+  score?: VulnerabilityScore;
```

Comment:
I think I'd rather this be `number` to more closely match the interface for `NxThreatIndicator`.  I know we can't cleanly lock it down to a discrete set of numbers like we do there, but I'm ok with that (as long as its checked at runtime).

Reply:
> I think I'd rather this be `number` to more closely match the interface for `NxThreatIndicator`. I know we can't cleanly lock it down to a discrete set of numbers like we do there, but I'm ok with that (as long as its checked at runtime).

I've updated the prop here: [bc1c460](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/bc1c460debb93d085727545318472bb9521685c2)